jekyll/_cci2/introduction-to-the-circleci-web-ui.adoc

```diff
@@ -0,0 +1,177 @@
+
+Projects associated with your organization will appear on the project's page. You will have the option to follow or unfollow projects that are already set up with CircleCI, or you can choose to set up a project with CircleCI. Following or setting up a project will add it to your dashboard. Unfollowing a project will remove it from your dashboard.
+
+Clicking the three dots (meatball menu) in the project row will allow you to open an individual project's configuration file, insights, and settings. If you are following a project, you will also see an option to unfollow the project from this menu. There is also a **Project Settings** button found on the dashboard of individual projects, which you can view by clicking on a project's name in the main dashboard.
+
+image::{{site.baseurl}}/assets/img/docs/web_ui_projects.png[Projects page]
+
+**Configuration File**: This option opens the project's configuration file inside a text editor within CircleCI's web UI. You can make changes to the config file and commit them. At the top of the text editor, you will see the project's branch you are committing the changes to (most often `main`).
+
+image::{{site.baseurl}}/assets/img/docs/web_ui_text_editor.png[Text editor in the web UI]
+
+**Insights**: This is a link to the insight's page, which is detailed <<#insights,below>>.
+
+[#project-settings]
+=== Project Settings
+
+**Overview**: Shows your project ID, as well as links to docs on how to set up certain features available to projects. You may also find the option to **Stop Building** on the overview page. This option halts builds, and will automatically _unfollow_ from all dashboards.
+
+**Triggers**: Allows you to define a timetable, parameter, and attribution to automatically run a pipeline when the criteria is met.
+
+**Advanced**: Toggle options on and off for: VCS status updates, build forked pull requests, pass secrets to builds from forked pull requests, only build pull requests, auto-cancel redundant workflows, free and open source, enable dynamic config using setup workflows. More information on these settings can be found on this page, as well as in our documentation.
+
+**Environment Variables**: Add or import environment variables to your jobs to keep sensitive data out of your repository.
+
+**SSH Keys**: See information related to your SSH keys, or set up new SSH keys. CircleCI uses deploy and user keys. There is an option to add additional SSH keys.
+
+**API Permissions**: Allows you to create API tokens, as well as revoke project-specific API tokens.
+
+**Jira Integrations**: Authenticate Jira and set up the Jira orb in your config file to integrate Jira into your projects.
+
+**Slack Integrations**: Authenticate Slack and set up the Slack orb in your config file to integrate Slack into your projects.
+
+**Insights Snapshot Badge**: A tool that allows you to generate a code snippet that will display a snapshot of your project's insights metrics in a README or other web document.
```

Comment:
I think I remember Rosie saying that while we don't capitalize feature names, Insights might be one exception..?

Reply:
Yes correct - Insights with a capital I